lib/service/supported_athletes/supported_nfl_athletes.dart

```diff
@@ -0,0 +1,91 @@
+class SupportedNFLAthletes {
+  //Example below
+  static const int _aLee = 3100;
+  static const int _aMack = 8485;
+  static const int _aPeterson = 4807;
+  static const int _aRodgers = 2593;
+  static const int _aSmith = 8459;
+  static const int _aWhitworth = 5099;
+  static const int _bHoyer = 8358;
+  static const int _bKern = 10067;
+  static const int _bRoethlisberger = 3807;
+  static const int _cCampbell = 3341;
+  static const int _cDaniel = 9902;
+  static const int _cHenne = 2405;
+  static const int _dAmendola = 9906;
+  static const int _dBrown = 3617;
+  static const int _dColquitt = 8243;
+  static const int _dJackson = 3943;
+  static const int _dMcCourty = 10978;
+  static const int _gGano = 10333;
+  static const int _jCook = 8534;
+  static const int _jFlacco = 611;
+  static const int _jJansen = 9162;
+  static const int _jJohnson = 6489;
+  static const int _jMcCourty = 8550;
+  static const int _jPeters = 8920;
+  static const int _kHuber = 8433;
+  static const int _mCox = 11008;
+  static const int _mCrosby = 1694;
+  static const int _mJenkins = 9196;
+  static const int _mLewis = 3061;
+  static const int _mPrater = 549;
+  static const int _mRyan = 732;
+  static const int _mSlater = 7651;
+  static const int _mStafford = 9038;
+  static const int _rFitzpatrick = 8283;
+  static const int _rGould = 1410;
+  static const int _rGronkowski = 10974;
+  static const int _rSuccop = 8750;
+  static const int _sKoch = 430;
+  static const int _tBrady = 4314;
+  static const int _tMorstead = 9201;
```

Comment:
I highly doubt we will be supporting all these athletes. Check in with Spiritual Kevin or with Robbie to confirm the list of NFL athletes